### PR TITLE
kazoo: Fixed connect to default vhost when used rabbitmq-c 0.8.0

### PR DESCRIPTION
**Pull request by sergey-safarov**

### CODE REVIEW

File: modules/kazoo/kz_amqp.c [hunk 1]
```diff
@@ -684,8 +684,6 @@ int kz_amqp_add_connection(modparam_t type, void* val)
 
     if(newConn->info.vhost == NULL) {
     	newConn->info.vhost = KZ_URL_ROOT;
-    } else if(newConn->info.vhost[0] == '/' && strlen(newConn->info.vhost) == 1) { // bug in amqp_parse_url ?
-    	newConn->info.vhost++;
```
Comment (lazedo) [inline modules/kazoo/kz_amqp.c]: this is needed for 0.5.2

File: pkg/kamailio/centos/7/kamailio.spec [hunk 2]
```diff
@@ -214,8 +214,8 @@ json string handling and RPC modules for Kamailio.
 %package    kazoo
 Summary:    Kazoo middle layer connector support for Kamailio
 Group:      System Environment/Daemons
-Requires:   libuuid, rabbitmq-c, json-c, libevent, kamailio = %ver
-BuildRequires:  libuuid-devel, rabbitmq-c-devel, json-c-devel, libevent-devel
+Requires:   libuuid, librabbitmq >= 0.8.0, json-c, libevent, kamailio = %ver
+BuildRequires:  libuuid-devel, librabbitmq-devel >= 0.8.0, json-c-devel, libevent-devel
```
Comment (lazedo) [inline pkg/kamailio/centos/7/kamailio.spec]: you can build with 0.5.2

File: modules/kazoo/kz_amqp.c (hunk 1 above)
Comment (sergey-safarov) [inline modules/kazoo/kz_amqp.c]: But this prevents to use rabbitmq-c lib 0.8.0
With this strings kazoo module cannot connect to rabbitmq-server

File: pkg/kamailio/centos/7/kamailio.spec (hunk 2 above)
Comment (sergey-safarov) [inline pkg/kamailio/centos/7/kamailio.spec]: Module can be build but later cannot be used if used rabbitmq-c lib 0.8.0
This PR is fix it

File: modules/kazoo/kz_amqp.c (hunk 1 above)
Comment (sergey-safarov) [inline modules/kazoo/kz_amqp.c]: I will rewrite this like it is done on commit https://github.com/kamailio/kamailio/commit/22e33ba64c3aad3b50e9ba649c5fa736462a30b4

**Comment by lazedo** [comment]
@sergey-safarov i would prefer to backport the related commit in master. i intent to create a PR to do that for this commit commit and a lot others if that's ok with you.

**Comment by sergey-safarov** [comment]
@lazedo this issue already fixed in master https://github.com/kamailio/kamailio/blob/master/src/modules/kazoo/kz_amqp.c#L92-L102

I see this PR not equal implementation in master branch.
Is required to update this PR?

**Comment by sergey-safarov** [comment]
@lazedo i goes to close this PR.
Please backport required features.

**Pull request closed without merging**